benchmark/run_phylomizer.sh

```diff
@@ -99,7 +99,7 @@ do
 
     # report the best aa substitution model and the -log likelihood of the tree
     best_tree=$(head -n1 input.tree.phyml.rank.ml)
-    best_model=$(echo ${best_tree} | cut -f1)
+    best_model=$(echo "${best_tree}" | cut -f1)
```

Comment:
The delimiter is `\t`, which is `cut`'s default delimiter. If one specifies it, the command will be as awkward as `cut -d$'\t' -f1`